### DISCUSSION TITLE
Metal developers must configure 2FA for all GitHub accounts

### DISCUSSION THREAD

**Discussion by tt-rkim** [tenstorrent/tt-metal#2755] [Contributor announcements]
GitHub is announcing all users must configure 2FA for their accounts by the end of 2023. They're slowly rolling it out to all organizations. The last date I heard from them is Friday, September 21st, 2023 for our organization, however they may push it back. We cannot request them to push it back as it's part of a large security initiative at GitHub.

2FA is an important security measure, as only one-factor passwords have clearly proven they're insufficient in the modern age of web security. Because of the sensitivity of our work, we may enable 2FA soon regardless of GitHub's deadlines and because of upcoming releases and open source initiative.

If you lose access to the organization, message me on Slack.

Administrators urgently need 2FA as soon as possible. If you're an administrator, you may lose admin privileges if you do not set up 2FA. The following people are administrators:

@davorchap @jvasilje @kkwong10 @boris-drazic @tt-aho @TT-billteng @DrJessop 

I cannot set up 2FA for you (obviously), so please don't ask me to set it up for you.

I personally use Google Authenticator, but you're welcome to use any method that GitHub allows you to.

Instructions:

https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa

@tenstorrent-metal/developers @tenstorrent-metal/external-developers

**Comment by tt-rkim** [comment]
I added another admin: @DrJessop

**Comment by TT-billteng** [comment]
FYI, I have 2FA set up

**Comment by tt-rkim** [comment]
This is a requirement for all members of this organization, regardless of your status.

Please refer to https://github.com/tenstorrent-metal/metal-internal-workflows/issues/109 for more information.
